packages/login/README.md

```diff
@@ -1,65 +1,57 @@
-# &lt;vaadin-login&gt;
+# @vaadin/login
 
-[Live Demo ↗](https://vaadin.com/components/vaadin-login/html-examples)
-|
-[API documentation ↗](https://vaadin.com/components/vaadin-login/html-api)
+A web component for creating a login form with two input fields and two buttons.
```

Comment:
```suggestion
A web component for displaying a configurable login form, either inline, or as an overlay.
```
Might want to remove `configurable` from the suggestion

Reply:
Thanks. I will omit `configurable` until we have custom content support.

Reply:
Done.